pkgs/development/libraries/chromaprint/default.nix

```diff
@@ -11,7 +12,13 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ cmake ];
 
-  buildInputs = [ boost ffmpeg ];
+  darwinInputs = [
```

Comment:
`darwinInputs` should not be declared in the scope of `stdenv.mkDerivation`, those declarations are passed to the build as environment variables, so declaring this here would make the darwinInputs a build dependency for all platforms.

Declare it in a `let...in` statement, or replace line 21 with `lib.optionals stdenv.isDarwin (with darwin.apple_sdk.frameworks; [Accelerate CoreGraphics CoreVideo zlib]);`.